src/index.ts

```diff
@@ -7,9 +7,7 @@ if (!process.env.JWT_PRIVATE_KEY) {
     logger.err('FATAL ERROR: jwtPrivateKey is not defined')
     process.exit(1);
 }
-const dbUrl = `${process.env.DB_URL}`
-    .replace('<USER>', process.env.DB_USER as string)
-    .replace('<PASSWORD>', process.env.DB_PASSWORD as string);
+const dbUrl = `${process.env.DATABASE_URL}`;
```

Comment:
ten url bierzemy z `production.env`? Jeśli tak, to: takiego pliku nie ma w repo, które widzi Heroku, bo każdy dev tworzy ten plik lokalnie u siebie :(

Reply:
Dlatego usunęłam wszystkie połączenia do production.env i zmienne ustawione są bezpośrednio w Heroku